test/controllers/images_controller_test.rb

```diff
@@ -4,6 +4,7 @@ class ImagesControllerTest < ActionController::TestCase
   setup do
     @image      = images(:one)
     @image.uuid = Foreman.uuid.to_s
+    Foreman::Model::Libvirt.any_instance.stubs(:image_exists?).returns(true)
```

Comment:
Thinking out loud: could this be updated to avoid fixtures and in particular, the stubbing of `any_instance` (which is considered a bad practice)?
```suggestion
    @image = FactoryBot.create(:libvirt_cr)
```

The answer is no because some items create new instances. Then it won't work. It may work to use `Fog.mock!` and `Fog.unmock!` to use dummy instances, but that may still not pass validation. You may need to update the code to ensure a dummy image exists and that's out of scope for now.